src/controllers/AnnotationModeController.js

```diff
@@ -267,6 +268,11 @@ class AnnotationModeController extends EventEmitter {
      */
     handleThreadEvents(thread, data) {
         switch (data.event) {
+            case THREAD_EVENT.save:
+            case THREAD_EVENT.cancel:
+                this.hasPendingThreads = false;
```

Comment:
Does setting this have any side effects for point/draw modes?

Reply:
This is used in the annotator to determine whether or not a mode is currently in the middle of creating an annotation (has an annotation pending). Doesn't have any side effects.